sys/kern/subr_vmem.c

```diff
@@ -1610,6 +1764,11 @@ bt_dump(const bt_t *bt, int (*pr)(const char *, ...))
 	    bt->bt_type, bt_type_string(bt->bt_type));
 }
 
+#endif /* defined(DDB) || defined(DIAGNOSTIC) */
+
+#if defined(DDB)
+#include <ddb/ddb.h>
+
```

Comment:
What's up with this change?

Reply:
I think this was caused by a mismerge and I missed it